src/clipper/ytc_settings.py

```diff
@@ -223,6 +223,9 @@ def _getVideoInfo(cs: ClipperState) -> Tuple[Dict[str, Any], Dict[str, Any]]:
         "youtube_include_dash_manifest": False,
     }
 
+    if settings["cookies"] != "":
+        ydl_opts["cookiefile"] = settings["cookies"]
```

Comment:
We may want to have the same cookiesfile used when using yt-dlp for downloading video subtitles. See  `src/clipper/ffmpeg_filter.py::getSubs`.

Reply:
Thanks for pointing that out! Fixed.